nixos/modules/system/activation/switch-to-configuration.py

```diff
@@ -0,0 +1,604 @@
+#! @python@
+
+import os.path
+import subprocess
+from syslog import openlog, syslog, LOG_NOTICE, LOG_ERR
+import sys
+
+out = "@out@"
+
+# To be robust against interruption, record what units need to be started etc.
+startListFile = "/run/systemd/start-list"
+restartListFile = "/run/systemd/restart-list"
+reloadListFile = "/run/systemd/reload-list"
+
+def die_with_usage():
+    print("""
```

Comment:
consider a function `error` which calls print with `file=stderr`
https://stackoverflow.com/questions/5574702/how-to-print-to-stderr-in-python